spec/workers/github_issue_worker_spec.rb

```diff
@@ -0,0 +1,8 @@
+RSpec.describe GithubIssueWorker do
+  describe '#perform' do
+    it 'Return correct handler class' do
+      handler = GithubIssueWorker.new.perform({ action: 'opened', issue: {} }.to_json)
+      expect(handler.class).to eq(GithubIssuesAuthor)
```

Comment:
nice, this will do

another way to do this would be something like
```ruby
  describe '#perform' do
    before :each do
      allow(GithubIssuesAuthor).to receive(: perform)
    end

    it 'Return correct handler class' do
      GithubIssueWorker.new.perform({ action: 'opened', issue: {} }.to_json)
      expect(GithubIssuesAuthor).to have_received(:perform).with('opened', {})
    end
  end
```

Reply:
Are you sure about that ? cause i'm getting "GithubIssuesAuthor does not implement: perform" i must be doing something wrong.

Reply:
should work...but yeah, no biggie...